lib/api/helpers.js

```diff
@@ -4,12 +4,22 @@ import { FlowRouter } from "meteor/kadira:flow-router-ssr";
 import { Shops } from "/lib/collections";
 
 /* eslint no-unused-vars: 0 */
-//
-//  TODO review this slugify import in lib/api/helpers
-//
+
+// Export Reaction using commonJS style for common libraries to use Reaction
+// https://docs.meteor.com/packages/modules.html#CommonJS
+let Reaction;
```

Comment:
I think this makes sense, @jshimko care to comment on this?  I think this make some thing easier, is this something we should also do for Schemas, Collections?